lstchain/image/muon/muon_analysis.py

```diff
@@ -193,7 +191,7 @@ def analyze_muon_event(subarray, event_id, image, geom, equivalent_focal_length,
          ])
 
     if candidate_clean_ring:
-        intensity_fitter = MuonIntensityFitter(subarray)
+        intensity_fitter = MuonIntensityFitter(subarray, hole_radius_m=0.308)
```

Comment:
can't we define this `hole_radius` as a global variable? It will not change for the LST

Reply:
Not worth, we will move to stage 1 in the next to next release.
I added that explicit argument just to suppress a warning (about MSTs and SSTs not being present). By default it will notice the only present telescope is an LST, and use the same value 0.308, but also throw the annoying warning with every instantiation.